### PR TITLE
Fix typo on TestNet README

### PR DESCRIPTION
**Pull request by linked0**
I didn't apply the spell-checked version of README in the last commit. So commit again.

### CODE REVIEW

File: devel/testnet/README.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ points to your own IP address.
 As you have read in the previous section, you can run a validator, but if the validator runs on
 a local computer without a publicly exposed ip, it cannot be reached from other validators.
 One option is to use a service like [ngrok](https://ngrok.com) to expose a local port to the public. If you only run
-a fullnode, not a validator, please skip this section.
+a full node, not a validator, please skip this section.
```
Comment (hewison-chris) [inline devel/testnet/README.md:146]: I think `fullnode` was correct here.

**Review by hewison-chris — CHANGES_REQUESTED** [review]

File: devel/testnet/README.md (hunk 1 above)
Comment (linked0) [reply devel/testnet/README.md:146]: Out of curiosity, can you explain the difference between them?

Comment (hewison-chris) [reply devel/testnet/README.md:146]: Looking at the rest of the `Readme` it does use `full node` and not `fullnode` (the name we use for the class in the code) so let's keep it consistent.

**Review by hewison-chris — APPROVED** [review]
LGTM

**Pull request merged**